gpflow/config/__config__.py

```diff
@@ -135,36 +172,50 @@ def config() -> Config:
 
 
 def default_int():
+    """Returns default integer type"""
     return config().int
 
 
 def default_float():
+    """Returns default float type"""
     return config().float
 
 
 def default_jitter():
+    """
+    The jitter is a constant that GPflow adds to diagonals of matrices
+    to achieve numerical stability of the system when conditional number of
+    matrices is large and therefore matrix is nearly singular.
+    """
     return config().jitter
 
 
 def default_positive_bijector():
+    """Type of bijector used for positive constraints: exp or softplus."""
     return config().positive_bijector
 
 
 def default_positive_minimum():
+    """Shift constant that GPflow adds to all positive constraints."""
     return config().positive_minimum
 
 
 def default_summary_fmt():
+    """Summary printing format."""
```

Comment:
```suggestion
    """Summary printing format as understood by tabulate ."""
```

Reply:
there is also the "notebook" special-case